### ISSUE TITLE
loading assets

### ISSUE THREAD

**Issue by mosesmuchemi** [prabhakar267/library-management-system#53]
How do you actually run the system? php artisan serve results in a blank run. When running directly via xampp the css does not load.

**Comment by prabhakar267** [comment]
@mosesmuchemi Did you follow all the instructions as mentioned in the README?

**Comment by mosesmuchemi** [comment]
Yes i did. I am using php 7.0 because it has mcrypt enabled by default. Installation went on smoothly.
